src/containers/language/language.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Select, MenuItem } from '@material-ui/core';
+import {
+  setToLocalStorage,
+  getFromLocalStorage
+} from '../../services/localstorage.service';
+import useStyles from './language.styless';
+import { changeLanguage } from '../../redux/language/language.actions';
+import { LANGUAGES_LIST } from '../../configs';
+
+const languageInLocalStorage = getFromLocalStorage('language') || 0;
+
+const Language = () => {
+  const styles = useStyles();
+  const dispatch = useDispatch();
+  dispatch(changeLanguage(languageInLocalStorage));
+
+  const handleChange = (e) => {
+    const targetValue = e.target.value;
+    if (targetValue === languageInLocalStorage) {
```

Comment:
Maybe it should be `!==` to perform this operation only when another language is selected. Do I miss something?

Reply:
I have simplified the code. this "if" statement is removed